spark/v4.0/spark/src/main/java/org/apache/iceberg/spark/actions/DeleteOrphanFilesSparkAction.java

```diff
@@ -300,7 +300,10 @@ private DeleteOrphanFiles.Result deleteFiles(Dataset<String> orphanFileDS) {
 
     LOG.info("Deleted {} orphan files", filesCount);
 
-    return ImmutableDeleteOrphanFiles.Result.builder().orphanFileLocations(orphanFileList).build();
+    return ImmutableDeleteOrphanFiles.Result.builder()
+        .orphanFileLocations(orphanFileList)
+        .deletedOrphanFilesCount(filesCount)
```

Comment:
`filesCount` is the total number of orphan file paths processed (i.e., attempted deletes) across all fileGroups, but it’s not guaranteed to mean “successfully deleted” (because deletes can fail).